opencog/atoms/bind/DefineLink.cc

```diff
@@ -84,4 +78,23 @@ DefineLink::DefineLink(Link &l)
 	init(l.getOutgoingSet());
 }
 
+Handle DefineLink::get_definition(const Handle& alias) {
+	// Get all DefineLinks associated with that alias, beware that it
+	// will also return DefineLink with that alias as definition body.
+    IncomingSet defs = alias->getIncomingSetByType(DEFINE_LINK);
+
+    // Return the first (supposedly unique) definition
```

Comment:
No, consider the following case
```atomese
DefineLink
   ConceptNode "B"
   ConceptNode "A"
DefineLink
   ConceptNode "C"
   ConceptNode "B"
```
although B is used only once as alias for something else, defs.size() would be 2, that is why I need to go through each of them and check if the alias matches.